x-pack/packages/observability/alert_details/src/components/alert_time_range_annotation.tsx

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import { RectAnnotation } from '@elastic/charts';
+import { i18n } from '@kbn/i18n';
+
+interface Props {
+  alertStart: number;
+  alertEnd?: number;
+  color: string;
+  id: string;
+}
+
+const RECT_ANNOTATION_TITLE = i18n.translate('observabilityAlertDetails.alertTimeRangeRect.title', {
+  defaultMessage: 'Active',
+});
+
+export function AlertTimeRangeAnnotation({ alertStart, alertEnd, color, id }: Props) {
+  return (
+    <RectAnnotation
+      id={id}
+      dataValues={[
+        {
+          coordinates: {
+            y0: 0,
+            x0: alertStart,
+            x1: alertEnd,
```

Comment:
Why do we need to add the `x1`?

Reply:
To only highlight the duration that the alert was firing.

Reply:
The chart will stop showing data when the alert is ended. So the end of the chart is the end of the alert. This is how we did it in APM. 
Are you showing data after the alert ended in the Metric chart?

Reply:
Yes, the calculation is similar to what we do for start: 20 mins after the alert was finished (unless `now` is before alertEnd + 20 mins)

Shall I merge my PR with the current implementation, and we discuss the end time of the chart with @maciejforcone  and @simianhacker ? (It should be an easy fix based on the decision)

Reply:
If the alert has ended, I agree we should do the same thing with the start and finish. This will come in handy when you're looking at a recovered alert; you'll be able to see the metric recover.